CHANGELOG.md

```diff
@@ -23,6 +23,15 @@ Dropping a requirement of a major version of a dependency is a new contract.
 ## [Unreleased]
 [Unreleased]: https://github.com/atlassian/aws-infrastructure/compare/release-2.6.0...master
 
+### Added
+- Gather Thread Dumps during Jira startup.
+- Add support for `infrastructure:4.9`.
```

Comment:
Which feature from 4.9.0 did we start supporting? We were compatible already.

Reply:
Thread dumps

Reply:
Ok, cool. We already cover that by "Gather Thread Dumps during Jira startup". Dependency bumps by themselves are not part of the contract. Only compatibility across major versions is a contract.
From the pragmatic perspective, it'd be nice to be able to drop support for 4.9.0 without breaking a contract. E.g. just like this PR drops support for 4.8.0.
It's fine to do that, because there should be no justifiable reason for consumers to avoid going from infrastructure:4.9.0 to infrastructure:4.10.0

Reply:
- [ ] Drop the 4.9.0 support contract